src/simulator_core/entities/assets/production_cluster.py

```diff
@@ -287,6 +277,9 @@ def write_to_output(self) -> None:
         - PROPERTY_PRESSURE_RETURN: The return pressure of the asset.
         - PROPERTY_MASSFLOW: The mass flow rate of the asset.
         """
+        if not self.simulation_performed():
+            return  # Remove this when you can perform a simulation
```

Comment:
The return raise seems odd?

Reply:
Is temporally otherwise app.py will not run